SendLog.py

```diff
@@ -31,38 +40,48 @@
 ENDING = 1
 DATA = 2
 NOLINE = -1
+BLOCK_IDENTIFICATION =- 2
+
 # Scilab dir, can't run absolute paths
-#SCI = "../scilab/"
-#SCI="../scilab-master/scilab/"
-SCI="../../scilab-master_5.5.2/"
+SCI = "../scilab_for_xcos/"
```

Comment:
Please do not use relative paths. This is highly OS specific and non-standard. Please use absolute paths and python ```os.path``` library to handle paths.

Please see, https://pymotw.com/2/ospath/